lib/good_bread/orders.ex

```diff
@@ -0,0 +1,13 @@
+defmodule GoodBread.Orders do
+  import Ecto.Query, warn: false
+  alias GoodBread.Repo
+  alias GoodBread.Orders.Shipment
+
+  # SHIPMENTS
+
+  def create_shipment(attrs \\ %{}) do
+    %Shipment{}
+    |> Shipment.changeset(attrs)
+    |> Repo.insert()
```

Comment:
This simply is creating a shipment record, not sure if I should create all the shipment items here are do that elsewhere